07-generic_bookworms/final/books/book_test.go

```diff
@@ -0,0 +1,80 @@
+package books_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"learngo-pockets/genericworms/books"
+	"learngo-pockets/genericworms/collectors"
+)
+
+func TestUnmarshalBook(t *testing.T) {
+	bookJson := []byte(`
+{
+  "author": "Sylvia Plath",
+  "title": "The Bell Jar"
+}`)
+
+	var book books.Book
+	err := json.Unmarshal(bookJson, &book)
+
+	assert.NoError(t, err, "unexpected error while unmarshalling book")
```

Comment:
This should be `require.NoError`, because the test should not continue from here.